bosh_cli/lib/cli/commands/deployment.rb

```diff
@@ -87,6 +87,43 @@ def perform
       recreate = !!options[:recreate]
       redact_diff = !!options[:redact_diff]
 
+
+      manifest = build_manifest
+      if manifest.hash['releases']
+        build_manifest.hash['releases'].each do |release|
+          unless release['url'].blank?
+            parsed_uri = URI.parse(release['url'])
+            case parsed_uri.scheme
+            when 'file'
+              run_nested_command "upload", "release", parsed_uri.path
+            when 'http', 'https'
+              err("No sha1 found for remote release `#{release['name']}'") if release['sha1'].blank?
+              run_nested_command "upload", "release", release['url'], "--sha1", "#{release['sha1']}"
+            else
+              err("Invalid url format for release `#{release['name']}' with url `#{release['url']}'")
```

Comment:
change to:

Invalid URL format for release `#{release['name']}' with URL `#{release['url']}'. Supported schemes: file, http, https.